pageserver/src/pgdatadir_mapping.rs

```diff
@@ -560,7 +560,7 @@ impl<'a> DatadirModification<'a> {
             slru_dir_to_key(SlruKind::MultiXactOffsets),
             empty_dir.clone(),
         );
-        self.put(slru_dir_to_key(SlruKind::Csn), empty_dir.clone());
+        self.put(slru_dir_to_key(SlruKind::Csn), empty_dir);
```

Comment:
Why? We can make this consistent with clog? That clones the empty_dir as well so we should either clone for both or not clone for either.

Reply:
This is suggested by the `clippy`, which is a rust linter.  The reason is that this is the last time `empty_dir` is used, so it is safe to just move it into the `put` function instead of making a copy.